utilities/_u-hide.scss

```diff
@@ -86,7 +86,7 @@ $u-hide-js-is-off-class:                           no-js !default;
 
 
   /**
-   * Form `legend`s need different treatment.
+   * Form `legend`'s need different treatment.
```

Comment:
This use of the apostrophe is ok, as the legend belongs to the form (and you should only have one legend).